src/Tizen.Wearable.CircularUI.Forms.Renderer/CircleImageRenderer.cs

```diff
@@ -37,7 +38,11 @@ protected override void OnElementChanged(ElementChangedEventArgs<CircleImage> e)
         {
             if (Control == null)
             {
-                _nativeParent = Platform.GetRenderer(Element.Parent).NativeView;
+                if (Element.Parent != null)
+                    _nativeParent = Platform.GetRenderer(Element.Parent).NativeView;
+                else
+                    _nativeParent = XForms.NativeParent;
+                _circleImage = new ElmSharp.EvasImage(_nativeParent);
```

Comment:
code duplicated.

                _circleImage = new ElmSharp.EvasImage(_nativeParent);
                _circleImage = new ElmSharp.EvasImage(_nativeParent);

Reply:
Please remove this line.

Reply:
removed